.github/workflows/build.yml

```diff
@@ -495,7 +495,9 @@ jobs:
         id: affected
         uses: ./.github/actions/affected
       - name: Set up NuGet sources
-        if: ${{ steps.affected.outputs.windows != '' && github.event_name == 'schedule' }}
+        # Temporarily disabled since builds using NuGet artifacts randomly fail
```

Comment:
since you mention "temporarily", should we track this with an issue somewhere and add a reference here?